### DISCUSSION TITLE
FAQ's and Troubleshooting V3

### DISCUSSION THREAD

**Discussion by akhrarovsaid** [payloadcms/payload#9555] [General]
# FAQ's and Troubleshooting V3

Hey Payload,

It would be nice if the docs could include a `FAQ's` section somewhere for the most common issues users encounter while building their shiny new Payload V3 app. 

In the meantime, I've compiled a list of the most common user issues I've encountered from over 130 pages of Discord discussions and threads for all to browse. I hope it helps users, contributors, and devs alike in getting to answers quickly. The intention here is for this to be a resource one can easily share a link to for a user in need.

If there are common questions you've noticed are missing, feel free to add them below or send me a DM on Discord and I'll add it here. The following list of Q & A's will appear in no particular order.

---

I've moved this to [it's own repo](https://github.com/akhrarovsaid/payload-v3-faqs/tree/main) to be able to link users to specific sections.

**Comment by matthijs166** [comment]
Why don't you make a pull request against the docs?
